src/intersection/intersection.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Creates an array of values that are in both the first and the second arrays.
+ *
+ * @param {Array} [array=[]] - The array to inspect.
+ * @param {Array} [values=[]] - The values to exclude.
+ * @returns {Array} Returns the new array of filtered values.
```

Comment:
'filter' is not easy to understand and arises question what is filtered? And filter is not mentioned in any other explanation in JSdoc.  instead 'intersected'.